js/src/main/scala/crystal/data/react/implicits/package.scala

```diff
@@ -1,9 +1,54 @@
 package crystal.data.react
 
-import japgolly.scalajs.react.Reusability
+import cats.implicits._
+import crystal.data._
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react._
 
 package object implicits {
+  implicit class PotRender[A](val pot: Pot[A]) extends AnyVal {
+    def renderPending(f: Long => VdomNode): VdomNode =
```

Comment:
I wonder if we should have a `fold` like method `fold(Long => VdomNode, Throwable => VdomNode, A => VdomNode)`

Reply:
We do! https://github.com/rpiaggio/crystal/blob/master/shared/src/main/scala/crystal/data/Pot.scala#L22

These separate render methods make component code more readable IMHO, especially if passed functions are several lines long.

Reply:
👍